test/gtest/ucp/test_ucp_sockaddr.cc

```diff
@@ -659,6 +653,51 @@ UCS_TEST_P(test_ucp_sockaddr, err_handle) {
 
 UCP_INSTANTIATE_ALL_TEST_CASE(test_ucp_sockaddr)
 
+class test_ucp_sockaddr_destroy_ep_on_err : public test_ucp_sockaddr {
+public:
+    void init() {
+        test_ucp_sockaddr::init();
+
+        ucp_ep_params_t params;
+        params.field_mask      = UCP_EP_PARAM_FIELD_ERR_HANDLING_MODE |
+                                 UCP_EP_PARAM_FIELD_ERR_HANDLER       |
+                                 UCP_EP_PARAM_FIELD_USER_DATA;
+        params.err_mode        = UCP_ERR_HANDLING_MODE_PEER;
+        params.err_handler.cb  = err_handler_cb;
+        params.err_handler.arg = NULL;
+        params.user_data       = &receiver();
+
+        receiver().add_server_ep_params(params, true);
```

Comment:
why do we need the helper function add_server_ep_params()?
can't we just override get_ep_params() ?

Reply:
helper function is able to merge different parameters and override/ignore duplications with info message, silent overriding by value can produce unexpected bugs